pkgs/applications/graphics/hdr-plus/0001-Update-to-Halide-10.0.0.patch

```diff
@@ -0,0 +1,183 @@
+From 5d2761528e2943b038b6ff2691591cc314c4d7d2 Mon Sep 17 00:00:00 2001
+From: OPNA2608 <christoph.neidahl@gmail.com>
+Date: Sun, 11 Oct 2020 13:11:07 +0200
+Subject: [PATCH] Update to Halide 10.0.0
+
+---
+ .github/workflows/main.yml | 16 +++++++---------
+ CMakeLists.txt             | 33 ++++++++++++++++++++++-----------
+ bin/HDRPlus.cpp            |  2 +-
+ bin/stack_frames.cpp       |  3 +--
+ src/halide_load_raw.h      |  1 -
+ 5 files changed, 31 insertions(+), 24 deletions(-)
+
+diff --git a/.github/workflows/main.yml b/.github/workflows/main.yml
+index 7e16b27..0db6c41 100644
+--- a/.github/workflows/main.yml
++++ b/.github/workflows/main.yml
```

Comment:
I don't think this GitHub Actions stuff matters to nix.

Reply:
Removed it from the patch.